### PR TITLE
Code formatting homework Asparuh

### PR DESCRIPTION
**Pull request by asparuh-krastev**

### CODE REVIEW

File: formatting-code/asparuh/JavaScript-code.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+b=navigator.appName;
+
+addScroll=false;
+if ((navigator.userAgent.indexOf('MSIE 5')>0) || (navigator.userAgent.indexOf('MSIE 6'))>0) {
```
Comment (victorlazov) [inline formatting-code/asparuh/JavaScript-code.js:4]: Line character count - 96.
Unnecessary brackets.
Spacing between operands and operators.

File: formatting-code/asparuh/Manager.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace League\CLImate\Settings;
+
+class Manager
+{
+    protected $setting = [];
```
Comment (victorlazov) [inline formatting-code/asparuh/Manager.php:7]: Drupal coding standards indent.

File: formatting-code/asparuh/Manager.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace League\CLImate\Settings;
+
+class Manager
+{
+    protected $setting = [];
+    
+    public function exists($name)
+    {
```
Comment (victorlazov) [inline formatting-code/asparuh/Manager.php:10]: Drup[al coding standards bracket positioning.

File: formatting-code/asparuh/JavaScript-code.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
+b=navigator.appName;
+
+addScroll=false;
+if ((navigator.userAgent.indexOf('MSIE 5')>0) || (navigator.userAgent.indexOf('MSIE 6'))>0) {
+    addScroll = true;
+}
+
+var off=0;
+var txt="";
```
Comment (anaseya) [inline formatting-code/asparuh/JavaScript-code.js:9]: var off & txt declared for what?

File: formatting-code/asparuh/JavaScript-code.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+            theLayer.style.pixelLeft=pX+10;
+            theLayer.style.pixelTop=pY+15;
+            theLayer.style.visibility='visible';
+        }
+    }
+}
+
+function HideTip()
+{
+    args=HideTip.arguments;
```
Comment (anaseya) [inline formatting-code/asparuh/JavaScript-code.js:70]: args declared for what? Then would be good to make an empty line above if statement :P

File: formatting-code/asparuh/JavaScript-code.js [hunk 6]
```diff
@@ -0,0 +1,121 @@
+b=navigator.appName;
+
+addScroll=false;
```
Comment (anaseya) [inline formatting-code/asparuh/JavaScript-code.js:3]: After line #8 declared variables have properly var statement, I think i should have the same for 'b' and 'addScroll'?

**Pull request merged**